src/main/java/internetShop/controller/ProductController.java

```diff
@@ -19,11 +22,11 @@ public ProductController(ProductService productService) {
         this.productService = productService;
```

Comment:
good practice to use 

@RequestMapping("/products")
class

@GetMapping("listProducts") or ("/deleteProduct") or something else
method()